src/gcp_scanner/scanner.py

```diff
@@ -176,49 +178,58 @@ def crawl_loop(initial_sa_tuples: List[Tuple[str, Credentials, List[str]]],
       if is_set(scan_config, 'service_accounts'):
         # Get service accounts
         project_service_accounts = crawl.get_service_accounts(
-            project_number, credentials)
+          project_number, credentials)
         project_result['service_accounts'] = project_service_accounts
 
       # Iterate over discovered service accounts by attempting impersonation
       project_result['service_account_edges'] = []
       updated_chain = chain_so_far + [sa_name]
 
       # Get GCP Compute Resources
-      compute_client = compute_client_for_credentials(credentials)
+      compute_service = ClientFactory.get_client(
+        'compute',
+      ).get_service(credentials)
+
       if is_set(scan_config, 'compute_instances'):
         project_result['compute_instances'] = crawl.get_compute_instances_names(
-                                                     project_id, compute_client)
+          project_id,
+          compute_service,
+        )
       if is_set(scan_config, 'compute_images'):
         project_result['compute_images'] = crawl.get_compute_images_names(
-                                                        project_id,
-                                                        compute_client)
+          project_id,
+          compute_service)
       if is_set(scan_config, 'machine_images'):
         project_result['machine_images'] = crawl.get_machine_images(
           project_id,
-          compute_client,
+          compute_service,
         )
       if is_set(scan_config, 'compute_disks'):
         project_result['compute_disks'] = crawl.get_compute_disks_names(
-                                                        project_id,
-                                                        compute_client)
+          project_id,
+          compute_service)
       if is_set(scan_config, 'static_ips'):
         project_result['static_ips'] = crawl.get_static_ips(project_id,
-                                                            compute_client)
+                                                            compute_service)
```

Comment:
This formatting seems odd as many of the others in this file have the args separated into new lines and left-aligned. I'm not too particular on which format we use, but we should be consistent.

Reply:
fixed